src/background/stores/assessment-store.ts

```diff
@@ -283,9 +283,9 @@ export class AssessmentStore extends BaseStore<IAssessmentStoreData> {
             return;
         }
 
-        payload.payloadBatch.forEach(payload => {
-            const testStepResult: ITestStepResult = assessmentData.generatedAssessmentInstancesMap[payload.selector].testStepResults[step];
-            testStepResult.isVisible = payload.isVisible;
+        payload.payloadBatch.forEach(batch => {
```

Comment:
`batch` to me means a group of things. The `payloadBatch` is an array of `UpdateInstanceVisibilityPayload` that we're iterating over. The arrow function parameter here is really one item or element of the array, perhaps using one of those terms might be clearer?

Reply:
Make sense.